.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,7 +10,7 @@ See also #23
 
 ## Checklist
 
-👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈
+👉 [How to sign off your commits with DCO](https://loopback.io/doc/en/contrib/code-contrib.html) 👈
```

Comment:
We should use `Sign off your commits with DCO (Developer Certificate of Origin)`. The items in the checklist are required actions instead of questions.